Tella/Data/Database/TellaDataBase.swift

```diff
@@ -191,6 +191,10 @@ class TellaDataBase {
         return try dataBaseHelper.delete(tableName: D.tServer,
                                          primarykeyValue: [KeyValue(key: D.cServerId, value: serverId)])
     }
+
+    func deleteAllServers() throws -> Int {
+        return try dataBaseHelper.deleteAll(tableName: D.tServer)
```

Comment:
@ValbuenaG Could you also delete report table and report files table (tReport and tReportInstanceVaultFile)